pkg/db/db.go

```diff
@@ -7,27 +7,31 @@ Copyright 2018
 Licensed under the Apache License, Version 2.0 (the "License")
 
 */
-package utils
+package db
 
 import (
 	"database/sql"
-	_ "github.com/lib/pq"
 	"fmt"
 	"time"
-	"github.com/pivotal-gss/gpmt2/log"
+
+	_ "github.com/lib/pq"
+	log "github.com/sirupsen/logrus"
 )
 
-var _db *sql.DB
+// globally defined, because the db object/connection should generally be in go
+// FIXME: Should this be exported?
+var db *sql.DB
 
+// MetalScanner is ... FIXME
 type MetalScanner struct {
 	valid bool
 	value interface{}
 }
```

Comment:
Its just a placeholder to keep the data and its type, since we don't know what would be the type of the column that is going to send to us when the user sends us a query.